masterlist.yaml

```
@@ -1089,3 +1089,73 @@ plugins:
   - name: 'OAAB_Tel Mora_Female Guards.ESP'
     url: [ 'https://www.nexusmods.com/morrowind/mods/46177/' ]
     after: [ 'OAAB_Tel Mora.esp' ]
+
+  ### Needs of Morrowind ###
+  - name: 'NOM 3.0.esp'
+    url: [ 'https://mw.modhistory.com/download-35-12114/' ]
```

That link breaks if it has a closing slash, you'll have to make an exception for some links.
```suggestion
    url: [ 'https://mw.modhistory.com/download-35-12114' ]
```